F4transkript/f4transkript.munki.recipe

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>Description</key>
+	<string>Downloads the latest version of f4transkript and imports into Munki.</string>
+	<key>Identifier</key>
+	<string>com.github.its-unibas.munki.f4transkript</string>
+	<key>Input</key>
+	<dict>
+		<key>MUNKI_REPO_SUBDIR</key>
+		<string>apps/f4transkript</string>
+		<key>NAME</key>
+		<string>f4transkript</string>
+		<key>pkginfo</key>
+		<dict>
+			<key>catalogs</key>
+			<array>
+				<string>testing</string>
+			</array>
+			<key>category</key>
+			<string>Creativity</string>
+			<key>description</key>
+			<string>f4transkript beschleunigt das eigene Abtippen von Audio- und Videodateien.</string>
+			<key>developer</key>
+			<string>audiotranskription.de</string>
+			<key>display_name</key>
+			<string>f4transkript</string>
+			<key>minimum_os_version</key>
+			<string>10.7.0</string>
+			<key>name</key>
+			<string>%NAME%</string>
+			<key>unattended_install</key>
+			<true/>
+			<key>blocking_applications</key>
+			<array>
+				<string>F5transkript</string>
+				<string>f4transkript</string>
+			</array>
+		</dict>
+	</dict>
+	<key>ParentRecipe</key>
+	<string>com.github.its-unibas.pkg.f4transkript</string>
```

Comment:
The download is a .app in a .zip Archive. This can be imported into Munki directly. We therefore don't need the .pkg recipe.

Reply:
Hallo Balz,
ich glaube laut der Dokumentation des munkiimporter Processor kann dieser nur .dmg oder .pkgs importieren.
Ich habe die recipes nun so angepasst, dass  in dem .download recipe nun die .zip entpackt wird, sodass man im .download recipe mit der entpackten .app direkt Code Signature Verification durchführen kann.
Im .munki recipe wird die .app kurz in ein .dmg verpackt und so in Munki importiert.
LG,
Oliver